schema.json

```diff
@@ -147,9 +144,5 @@
         "oneOf": { "$ref": "#/definitions/schemaArray" },
         "not": { "$ref": "#" }
     },
-    "dependencies": {
-        "exclusiveMaximum": [ "maximum" ],
-        "exclusiveMinimum": [ "minimum" ]
-    },
     "default": {}
```

Comment:
It could be:

```json
"dependencies": {
  "exclusiveMaximum": {
    "anyOf": [
      {
        "properties": {
          "exclusiveMaximum": { "type": "number" }
          "maximum": false
        }
      },
      {
        "properties": {
          "exclusiveMaximum": { "type": "boolean" }
          "maximum": true
        }
      }
    ]
  }
}
```